VERSION

```diff
@@ -1 +1 @@
-3.10.0
+3.10.0.master
```

Comment:
I guess we have to ask ourselves if we want to follow existing methods and use a dash -- https://github.com/theforeman/foreman-installer/blob/develop/VERSION#L1

Reply:
I'm fine to stick with convention. Apologies, I just looked at what [katello-installer-base.spec](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/katello/katello-installer-base/katello-installer-base.spec#L15) looked for.